bin/post_process_sce.R

```diff
@@ -72,28 +65,20 @@ if(!(stringr::str_ends(opt$output_sce_file, ".rds"))){
 # read in input rds file
 sce <- readr::read_rds(opt$input_sce_file)
 
-# check that prob compromised cutoff is between 0-1
-if(!dplyr::between(opt$prob_compromised_cutoff, 0, 1)){
-  stop("--prob_compromised_cutoff must be a number between 0 to 1")
-}
-
 # create ccdl_filter column
 if(all(is.na(sce$prob_compromised))){
```

Comment:
Since we are using `miQC_pass` now, I think we should test on that? (See also prior comment)
```suggestion
if(all(is.na(sce$miQC_pass))){
```